.github/PULL_REQUEST_TEMPLATE/new_connector.md

```diff
@@ -0,0 +1,24 @@
+## Checklist
+- [ ] Ran Linter
+- [ ] Catalog tests passing
+- [ ] Created PR from non-main branch
```

Comment:
This third item is probably not needed, Github wouldn't have let you create a PR from a main branch.

Reply:
This was for forks, but is irrelevant now